### DISCUSSION TITLE
Tele-Wifi#Downtime

### DISCUSSION THREAD

**Discussion by pbponde** [arendst/Tasmota#12765] [Q&A] [answered]
Hi!
I'm trying to create a rule that reboots the device after 5min downtime, but nothing happens. What am I doing wrong?

Rule1 ON Tele-Wifi#Downtime>300 DO Restart1 ENDON
Rule1 1

**Comment by barbudor** [comment, marked as answer]
Wifi#Downtime is not an integer.

I would suggest to use 'wifi#disconnected' / 'wifi#connected' events along with a rule timer

**Reply by sfromis** [reply]
You can to a limited extent use string matching triggers to react to length of downtime
`on wifi#Downtime$^T00:00 do Restart 1 endon`
This one will trigger if the downtime does not contain the string `T00:00`, indicating that downtime has exceeded one minute.

Note that the space in the restart command is very important for it to actually do a restart.

**Reply by pbponde** [reply]
Thanks! This kooks to be the best choice!
